### DISCUSSION TITLE
live grep while ignoring comments?

### DISCUSSION THREAD

**Discussion by erwin** [NvChad/NvChad#1669] [Q&A]
Curious if anyone has found a way to do something like the Telescope live grep, but while ignoring comments.

I saw there's a rust tool built on top of treesitter called "tree grep" that may be useful...

https://github.com/BrianHicks/tree-grepper

Curious if anyone has found a nice way to do this in nvim that, and how it works with the rest of your nvchad config?

**Comment by siduck** [comment]
hmm this isnt a nvchad issue! you can ask this in the telescope community, we just merely use its live_grep command
